app/components/error_text/index.tsx

```diff
@@ -14,7 +14,7 @@ type ErrorProps = {
     error: ClientErrorWithIntl | string;
     testID?: string;
     textStyle?: StyleProp<ViewStyle> | StyleProp<TextStyle>
-    theme: Theme;
+    theme?: Theme;
```

Comment:
why is this optional? as far as the code goes it doesn't look optional at all

Reply:
For Forgot Password and MFA, we are not passing a `theme` prop.

Reply:
we should pass it tough.. the theme there can be the default theme based on the `appearance` let me know if you need help with that

Reply:
You mean to default theme to the value return from `getThemeFromState()` ?